lib/_http_incoming.js

```diff
@@ -315,6 +315,8 @@ IncomingMessage.prototype._dump = function _dump() {
   if (!this._dumped) {
     this._dumped = true;
     this.resume();
+    // Flush the existing buffered data
+    this._readableState._flush();
```

Comment:
@mcollina and @mscdex Thanks for the comments. If the access of `_*State` would be removed, I guess, at least there should be new APIs in `Readable` for those `_*State` access. @mcollina Should I move the `_flush()` to `Readable`. The flush like API may be needed in some other cases. I think this is related to `request.abort()`. Currently, it doesn't work in some platforms.